app/services/push_notification/account_reinstated_event.rb

```diff
@@ -0,0 +1,27 @@
+module PushNotification
+  class AccountReinstatedEvent
+    EVENT_TYPE = 'https://schemas.openid.net/secevent/risc/event-type/account-reinstated'.freeze
```

Comment:
this event is not a standard event. Here's the list of events in the spec:

https://openid.net/specs/openid-risc-event-types-1_0-ID1.html

I think that "reinstated" maps to `account-enabled` and "suspended" maps to `account-disabled`?

But if we're not sure, we should invent our own at the `schemas.login.gov` domain like we have for others: https://developers.login.gov/security-events/#authorization-fraud-detected